packages/http/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@stoplight/prism-http",
-  "version": "0.0.0",
+  "version": "3.0.0-pre",
```

Comment:
Hmm, do we need to start from v3?
I mean `@stoplight/prism-http` has never been release so it should start from 1 right?
IIRC lerna with release ``prism-http-1.0.0` anyway so should be no collision with existing tags (correct?)

This obviously applies to all packages.

Reply:
Ah, this is a good point. Technically we have never published this on npm so it's effectively a 1.0 — but on the other hand it's branded as "Prism 3.0"; 

@philsturgeon how do we want to go with the versioning? 1.x or 3.x?

I *think* we should go with 3.0; we can theoretically use the 2.x tags on npm to download the old prims binary so that we offer a "streamlined" experience, but I'm open to alternatives.

Reply:
Ok, a crazy idea, maybe 3.0 for prism-cli and 1.0 for the framework stuff? 

Meh :p 

Or just go with 3 :)

Reply:
In theory it's possible to have different versions but that might complicate the things a little bit; unless it's an hard requirement, I'd stick with the current 3.x stack (or start again with 1.x)

Reply:
Yup stick to 3.x, let's not be purists.